lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider.rb

```diff
@@ -8,12 +8,6 @@ class LighthouseSupplementalDocumentUploadProvider
 
   STATSD_PROVIDER_METRIC = 'lighthouse_supplemental_document_upload_provider'
 
-  # Maps VA's internal Document Types to the correct document_type attribute for a Lighthouse526DocumentUpload polling
-  # record. We need this to create a valid polling record
-  POLLING_DOCUMENT_TYPES = {
-    'L023' => Lighthouse526DocumentUpload::BDD_INSTRUCTIONS_DOCUMENT_TYPE
-  }.freeze
```

Comment:
This was mistakenly left behind when implementing https://github.com/department-of-veterans-affairs/vets-api/pull/19117